deploy/examples/create-external-cluster-resources.py

```diff
@@ -1152,6 +1152,12 @@ def create_rgw_admin_ops_user(self):
                     "info",
                     "--uid",
                     self.EXTERNAL_RGW_ADMIN_OPS_USER_NAME,
+                    "--rgw-realm",
+                    self._arg_parser.rgw_realm_name,
```

Comment:
If these parameters are not specified, will the command still succeed with defaults (as long as the defaults are valid)? Or if these args are now required, perhaps we would need a doc update.

Reply:
The command succeeds is what I have checked.
And it makes it in `default`  discussed here https://github.com/rook/rook/pull/12037#discussion_r1163333816.
@alimaredia to keep me honest.